src/longopt/grdproject_inc.h

```diff
@@ -25,32 +25,20 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'D', "",
-	          "",                  "",
+	{ 0, 'C', "center",            "", "", "", "", GMT_TP_STANDARD },
+	{ '/', 'D', "increment|inc|spacing",
 	          "",                  "",
+	          "e,n",               "exact,number",
 		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'F', "",
-	          "",                  "",
+	{ 0, 'E', "dpi",               "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'F', "one2one|one2one_scale",
+	          "c,i,p,e,f,k,M,n,u", "cm,inch,point,meter,foot,km,smile,nmile,ussft",
```

Comment:
This may be very late in the game to ask, but I think we should be more flexible here, since it would be easy to forget the exact syntax. Can we do something like:
"cm,inch|in,point|pt,meter|m,foot|ft,km,smile|mi,nmile|nm,ussft"

I'm not sure that a single letter "m" would work. To be checked.
Also this should then be used consistently in other modules.